PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDebuggerRunner.java

```diff
@@ -152,7 +152,7 @@ protected void execute(@NotNull ExecutionEnvironment environment, @Nullable Call
                                     "Error : Spark batch Job remote debug failed, got exception: " + errorMessage);
 
                             postEventProperty.put("IsSubmitSucceed", "false");
-                            postEventProperty.put("SubmitFailedReason", errorMessage.substring(0, 50));
+                            postEventProperty.put("SubmitFailedReason", errorMessage.substring(0, errorMessage.length() > 50 ?  50 : errorMessage.length()));
```

Comment:
Maybe we should use a util class to post the event  and set the max size `50` as a `constant`.